docs/source/qcarchive_overview.rst

```diff
@@ -0,0 +1,128 @@
+QCArchive Design
+==================
+
+The QCArchive software ecosystem consists of a series of Python modules that
+can either be used together or are useful standalone pieces to the
+computational molecular sciences community. This ecosystem is constructed to
+be used at single-user, small group, and multi-PI levels while retaining
+the ability to scale up to the needs of an entire community of scientist.
+
+In each case, it is expected only a small number of users are required to
+understand the entire software stack and the primary interaction with the
+QCArchive ecosystem will be through the user front-end (QCPortal). After the
+persistence server instance (QCFractal) is instantiated with a distributed
+workflow system and compute the server should be able to maintain
+itself without user intervention. A diagram of how the ecosystem works in
+concert can be seen below:
+
+
+
+.. image:: media/boxology_overview.jpg
+   :width: 800px
+   :alt: QCArchive project structure
+   :align: center
+
+1) QCPortal
+++++++++++++
+
+- GitHub: |QCPortalBadge|_
+- Hardware: Laptop
+- Actor: User
+- Primary Developer: MolSSI
+
+.. |QCPortalBadge| image:: https://img.shields.io/github/stars/MolSSI/QCPortal.svg?style=social&label=Stars
+.. _QCPortalBadge: https://github.com/MolSSI/QCPortal
+
+QCPortal provides a Python-based user front-end experience for users who are
+interested in exploring data and executing new tasks. Exploration of computed
+data is augmented by the ability to generate graphs quickly and other
+representations to view the data in Jupyter notebooks and high-level
+abstractions are used to view and manipulate many individual tasks
+simultaneously. Querying of data and submission of new tasks occurs over the
+QCFractal REST API.
+
+
+
+2) QCFractal
+++++++++++++
+
+- GitHub: |QCFractalBadge|_
+- Hardware: Persistent Server
+- Actor: Power User
+- Primary Developer: MolSSI
+
+.. |QCFractalBadge| image:: https://img.shields.io/github/stars/MolSSI/QCFractal.svg?style=social&label=Stars
+.. _QCFractalBadge: https://github.com/MolSSI/QCFractal
+
+QCFractal is the primary persistent server that has several main duties:
```

Comment:
I think it would be worth adding in here that QCFractal serves as the server which QCPortal communicates with. Its a bit redundant with the QCPortal entry above, but I think that is okay for this type of think which is supposed to discuss connectivity. And if a power user starts with the QCFractal entry, they may not know that the QCPortal entry pointed here.